### PR TITLE
Introduce initial empty vscode extension

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce initial empty vscode extension created from yo tools.

ONE-vscode-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
From draft #14

**Comment by mhs4670go** [comment]
I think `package-lock.json` file should be added as well. Please see https://github.com/Samsung/ONE-vscode/pull/17/files#r698430764.

**Comment by seanshpark** [comment]
I'll rebase this after #17 lands

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "lib": [
+      "es6"
+    ],
+    "sourceMap": true,
+    "rootDir": "src",
+    "strict": true   /* enable all strict type-checking options */
+    /* Additional Checks */
+    // "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
+    // "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
+    // "noUnusedParameters": true,  /* Report errors on unused parameters. */
```
Comment (mhs4670go) [inline tsconfig.json:15]: How about turn on this check? Or Is it going to be enabled later?

Comment (seanshpark) [reply tsconfig.json:15]: Yes, later :)

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]

**Comment by seanshpark** [comment]
Added `package-lock.json` file too. Please take a look again :)

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]

**Pull request merged**